experimental-design/README.adoc

```diff
@@ -141,5 +141,71 @@ Some search engines as MaxQuant use the Fraction Group to perform better statist
 - Comment [Fraction Group]: Some Quantitative tools use the Fraction Group to know which fractions belong together. In MaxQuant the Fraction Group is called "Experiment".
 
 
+[[encoding-MSRun-technical-details]]
+== MSRun technical details properties
 
+We RECOMMEND to encode some of the technical parameters of the mass spectrometry experiment as Comments (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a Comment in SDRF]) including the following parameters:
 
+- Protein Modifications <<encoding-protein-modifications>>
+- Precursor and Fragment mass tolerances <<encoding-tolerances>>
+- Enzyme <<encoding-enzymes>>
+
+[[encoding-protein-modifications]]
+=== Protein Modifications
+
+Sample modifications (PTMs) are originated from multiple sources: **artifacts modifications**, **isotope labeling**, adducts that present as PTMs (e.g . sodium) or the most **biologically relevant** the wide variety of chemical modifications after translation. The most common and widely studied post translational modifications include phosphorylation and glycosylation. Many of these post-translational modifications are critical to the protein's function.
+
+The current specification RECOMMEND to provide Sample modifications including the Amino acid affected, if is Variable or Static (Fixed) and other properties such as mass shift and position (e.g. anywhere in the sequence).
+
+The RECOMMENDED name of the column for sample modification parameters is:
+
+  Comment [modification parameters]
+
+.. NOTE: The `modification parameters` is the name of the ontology term https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001055[MS:1001055]
+
+For each modification, we will capture different properties in a `key=value` pair structure including name, position, etc. All the possible features available for modification parameters:
+
+
+|===
+|Property |Key |Example | Required/Optional |Comment
+
+|Name of the modification| NM | NM=Acetylation | Required | * Name of the modification, for custom modifications can be a name defined by the user.
+
+|Database Accession| AC | AC=UNIMOD:1 | Optional | Accession in an external database UNIMOD or PSI-MOD supported.
+
+|Chemical Formula  | CF | CF=H(2)C(2)O| Optional | This is the chemical formula of the added or removed atoms. For the formula composition please follow the guidelines from http://www.unimod.org/names.html[UNIMOD]
+
+|Modification type | MT | MT=Fixed | Required | This specifies which modification group the modification should be included with. Choose from the following options: [Fixed, Variable, Custom].
```

Comment:
Could add an option: "Annotated Positions" to cover the case where a mod is selectively applied

Reply:
Can you give an example @trishorts.

Reply:
I don't know how to give you an example. When we do search in MetaMorpheus, we download database will many PTMs annotated at many positions. And we look for them all. So, we'll see many phosphoserine. We don't search variable phospho and we don't search fixed phospho, rather we search annotated phospho......

Reply:
Well, all those modifications are not provided by the user but mainly a search engine option.

Reply:
.......but that is how the data were searched. And, if you want a clear record of how the data was searched so that others can reproduce it in the future then you kind of have to know that fact. We're talking about lots of psms and PTMs. Something like 5-10% of peptides.

Reply:
@trishorts sorry, I got your point.

Reply:
no apologies and no need to get this perfect. I'd rather have something that works pretty good for most things than nothing at all in hopes that we can find some perfect solution someday. This is already a most excellent beginning.